Library/Tracking/KSRAnalytics.swift

```diff
@@ -703,11 +703,13 @@ public final class KSRAnalytics {
   public func trackAddOnsContinueButtonClicked(
     project: Project,
     reward: Reward,
+    checkoutData: CheckoutPropertiesData,
     refTag: RefTag?
   ) {
     let props = projectProperties(from: project, loggedInUser: self.loggedInUser)
       .withAllValuesFrom(pledgeProperties(from: reward))
-      .withAllValuesFrom(contextProperties())
+      .withAllValuesFrom(contextProperties(ctaContext: .addOnsContinue))
+      .withAllValuesFrom(checkoutProperties(from: checkoutData, and: reward))
 
     self.track(
       event: ApprovedEvent.addOnsContinueButtonClicked.rawValue,
```

Comment:
The ticket is requesting the name of the event to be changed to "CTA Clicked". I think the best practice moving forward will be to delete the event being renamed from `ApprovedEvent`. In Afees' current PR he will be creating a new enum with the generic event names and once you rebase/pull his changes you can add `CTA Clicked` into there and replace the event on line 715.